### PR TITLE
Fix  support + drop fixtures/directories support

### PR DESCRIPTION
**Pull request by Cethy**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes #27 
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | yes
| Tests added   | not really 
| Doc added     | updated 

- Fixing the `doctrine.fixture.orm` support ;
- Drop `fixtures` & `directories` options since they are dropped from `DoctrineFixturesBundle` ;

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,12 +3,22 @@ Doctrine Fixtures Extension for Behat
...
 composer require "behat-extension/doctrine-data-fixtures-extension"
 ```
 
+```php
+<?php # config/bundles.php
+return [
+    \BehatExtension\DoctrineDataFixturesExtension\Bundle\BehatDoctrineDataFixturesExtensionBundle::class => ['test' => true],
+];
+```
+**@todo** : flex recipe ?
```
Comment (Spomky) [inline README.md:20]: Agreed :+1:

File: README.md [hunk 2]
```diff
@@ -20,38 +30,13 @@ default:
...
-    BehatExtension\DoctrineDataFixturesExtension\Extension:
-      lifetime: 'feature'
-      directories:
-        - '/project/src/AcmeAnalytics/Tests/DataFixtures/ORM'
-      fixtures:
-        - 'Acme\StoreBundle\DataFixture\ORM\Categories'
-        - 'Acme\StoreBundle\DataFixture\ORM\Apps'
-        - 'Acme\VendorBundle\DataFixture\ORM\Vendors'
-```
+This extension will load every fixtures declared as services and tagged with `doctrine.fixture.orm`.
```
Comment (Spomky) [inline README.md:39]: The problem here is that the exension won't work anymore for non-Symfony applications

File: tests/DemoBundle/Resources/config/services.php [hunk 3]
```diff
@@ -26,10 +26,10 @@
     $container->load(
         'BehatExtension\\DoctrineDataFixturesExtension\\Tests\\DemoBundle\\Tests\\DataFixtures\\',
         __DIR__.'/../../Tests/DataFixtures/*'
-    )->private();
+    )->tag('doctrine.fixture.orm')->private();
```
Comment (Spomky) [inline tests/DemoBundle/Resources/config/services.php:29]: Is it needed as the autoconfiguration is enabled?

**Comment by Cethy** [comment]
Closing in favor of https://github.com/BehatExtension/DoctrineDataFixturesExtension/pull/29 ;

**Pull request closed without merging**